tgui/packages/tgui/interfaces/MafiaPanel.js

```diff
@@ -0,0 +1,161 @@
+            anyone! It will runtime the game to death! <br />
+            <Button
+              icon="arrow-right"
+              onClick={() => act("next_phase")}>Next Phase
+            </Button>
+            <Button
+              icon="home"
+              onClick={() => act("players_home")}>Send All Players Home
+            </Button>
+            <Button
+              icon="radiation"
+              onClick={() => act("new_game")}>New Game
+            </Button>
+          </Section>
+        )}
+        <Section title="Players">
+          <LabeledList>
+            {!!players && players.map(player => { return (
+              <LabeledList.Item
+                className="candystripe"
+                key={player.ref}
+                label={player.name}>
+                {!player.alive && (<Box color="red">DEAD</Box>)}
+                {player.votes !== undefined && !!player.alive
+                && (<Fragment>Votes : {player.votes} </Fragment>)}
+                {
+                  !!player.actions && player.actions.map(action => {
+                    return (
+                      <Button
+                        key={action}
+                        onClick={
+                        // eslint-disable-next-line indent
+      () => act("mf_targ_action", { atype: action, target: player.ref })
+                        }>
+                        {action}
+                      </Button>); })
+                }
+              </LabeledList.Item>);
+            })}
+          </LabeledList>
+        </Section>
+        {!!judgement_phase && (
+          <Section title="JUDGEMENT">
+            <Flex justify="space-around">
+              <Button
+                icon="smile-beam"
+                color="good"
+                onClick={() => act("vote_innocent")}>INNOCENT!
+              </Button>
+              Use these buttons to vote the accused innocent or guilty!
+              <Button
+                icon="angry"
+                color="bad"
+                onClick={() => act("vote_guilty")}>GUILTY!
+              </Button>
+            </Flex>
+          </Section>
+        )}
+        <Flex mt={1} spacing={1}>
+          <Flex.Item grow={1} basis={0}>
+            <Section
+              title="Roles">
+              <Table>
+                {!!all_roles && all_roles.map(r => (
+                  <Table.Row key={r}>
+                    <Table.Cell bold>
+                      <Flex justify="space-between">
+                        {r}
+                        <Button
+                          content="?"
+                          onClick={() => act("mf_lookup", { atype: r })}
+                        />
+                      </Flex>
+                    </Table.Cell>
+                  </Table.Row>
+                ))}
+              </Table>
+            </Section>
+          </Flex.Item>
+          <Flex.Item grow={2} basis={0}>
+            <Section
+              title="Notes">
+              <Table>
```

Comment:
You don't need a table here at all, just return plain boxes from the `roleinfo` loop.